tabs/_common.py

```
@@ -1,166 +1,169 @@
-# No imports needed - this module provides pure data functions
+from typing import TYPE_CHECKING, Dict, Optional, Any
```

Ruff (0.14.10)

`typing.Dict` is deprecated, use `dict` instead

(UP035)